packages/client/test/index.spec.ts

```diff
@@ -8,7 +8,10 @@ import { expect } from 'aegir/chai'
 import { createIPNSRecord, marshalIPNSRecord } from 'ipns'
 import all from 'it-all'
 import { CID } from 'multiformats/cid'
-import { createDelegatedRoutingV1HttpApiClient, type DelegatedRoutingV1HttpApiClient } from '../src/index.js'
+import {
+  createDelegatedRoutingV1HttpApiClient,
+  type DelegatedRoutingV1HttpApiClient
+} from '../src/index.js'
```

Comment:
another unnecessary change

```suggestion
import { createDelegatedRoutingV1HttpApiClient, type DelegatedRoutingV1HttpApiClient } from '../src/index.js'
```